client/blocks/signup-form/index.jsx

```diff
@@ -1041,14 +1040,11 @@ class SignupForm extends Component {
 
 		if ( isBlazePro ) {
 			return (
-				<div>
-					<LoggedOutFormLinks>
-						<span>{ this.props.translate( 'Already have an account?' ) }&nbsp;</span>
-						<LoggedOutFormLinkItem href={ this.getLoginLink() }>
-							{ this.props.translate( 'Log in here' ) }
-						</LoggedOutFormLinkItem>
-					</LoggedOutFormLinks>
-				</div>
+				<p className="signup-form__login-link">
+					{ this.props.translate( 'Already have an account? {{link}}Log in here{{/link}}.', {
```

Comment:
Fixes issues with the translation, as the complete string is now able to be translated.